demos/demo_par_decimator/XCORE-AI-EXPLORER.xn

```diff
@@ -0,0 +1,124 @@
+            <Port Location="XS1_PORT_1O"  Name="PORT_I2C_SDA"/>
+
+            <Port Location="XS1_PORT_4C" Name="PORT_LEDS"/>
+            <Port Location="XS1_PORT_4D" Name="PORT_BUTTONS"/>
+
+            <!-- <Port Location="XS1_PORT_1I"  Name="WIFI_WIRQ"/> -->
+            <!-- <Port Location="XS1_PORT_1J"  Name="WIFI_MOSI"/> -->
+            <!-- <Port Location="XS1_PORT_4E"  Name="WIFI_WUP_RST_N"/> -->
+            <!-- <Port Location="XS1_PORT_4F"  Name="WIFI_CS_N"/> -->
+            <!-- <Port Location="XS1_PORT_1L"  Name="WIFI_CLK"/> -->
+            <!-- <Port Location="XS1_PORT_1M"  Name="WIFI_MISO"/> -->
+
+            <!--
+              NOTE: Depending on app_config.h these ports/pins may be utilized:
+              * PORT_MCLK_IN_OUT, XS1_PORT_1D (Tied to both Tile 0 and 1's PORT_1D via R90 & R79)
+              * PORT_PDM_CLK, XS1_PORT_1A, X0D00, J14 - Pin 2
+              * PORT_PDM_DATA, XS1_PORT_4C, X0D14,X0D15,X0D20,X0D21, J14 - Pin 3,5,12,14
+              * PORT_PDM_DATA, XS1_PORT_4D, X0D16..X0D19, J14 - Pin 6,7,10,11
+              * PORT_PDM_DATA, XS1_PORT_8B (Same pins as 4C and 4D above)
+              * Please take note that the LEDs, Buttons, and USB on this board may not be usable in
+              * certain configurations.
+            -->
+          </Tile>
+          <Tile Number="1" Reference="tile[1]">
+            <!--
+              NOTE: Depending on app_config.h these ports/pins may be utilized:
+              * PORT_PDM_CLK, XS1_PORT_1M, X1D36, J10 - Pin 2
+              * PORT_PDM_DATA, XS1_PORT_4A, X1D02,X1D03,X1D08,X1D09 | J10 - Pin 2,5,12
+              * PORT_PDM_DATA, XS1_PORT_4B, X1D04..X1D07 | J10 - Pin 6,7,10,11
+              * PORT_PDM_DATA, XS1_PORT_8A (Same pins as 4A and 4B above)
+              * PORT_CODEC_RST_N, XS1_PORT_1O, X1D38 | J10 - Pin 15
+              * PORT_CODEC_RST_N, XS1_PORT_4A, X1D08 | J10 - Pin 12
+            -->
+
+            <!-- Audio ports -->
+            <Port Location="XS1_PORT_1D" Name="PORT_MCLK_IN_OUT"/>
+            <Port Location="XS1_PORT_1C" Name="PORT_I2S_BCLK"/>
+            <Port Location="XS1_PORT_1B" Name="PORT_I2S_LRCLK"/>
+            <Port Location="XS1_PORT_1A" Name="I2S_DATA_IN"/> <!-- I2S_DAC_DATA -->
+            <Port Location="XS1_PORT_1N" Name="I2S_DATA_OUT"/> <!-- I2S_ADC_DATA -->
+          </Tile>
+        </Node>
+      </Nodes>
+    </Package>
+  </Packages>
+  <Nodes>
+    <Node Id="2" Type="device:" RoutingId="0x8000">
+      <Service Id="0" Proto="xscope_host_data(chanend c);">
+        <Chanend Identifier="c" end="3"/>
+      </Service>
+    </Node>
+  </Nodes>
+  <Links>
+    <Link Encoding="2wire" Delays="5clk" Flags="XSCOPE">
+      <LinkEndpoint NodeId="0" Link="XL0"/>
+      <LinkEndpoint NodeId="2" Chanend="1"/>
+    </Link>
+  </Links>
+  <ExternalDevices>
+    <Device NodeId="0" Tile="0" Class="SQIFlash" Name="bootFlash" Type="S25FL116K" PageSize="256" SectorSize="4096" NumPages="16384">
```

Comment:
I'm a bit surprised at the inclusion of the Type, PageSize, SectorSize and NumPages arguments.  I thought that recent changes Tools 15.2 meant that we no longer needed these arguments and that their inclusion caused problems.  However, my notoriously bad memory means that I am not sure of the facts, and I see that the other .xn files in this repository look similar to this one as far as this line goes.

Reply:
You are correct - this shows it https://github.com/xmos/sw_xvf3800/commit/4349ede0fd3b0527462de01da0402cdcc5322aa9

I will fix!